src/WebJobs.Extensions/Extensions/Timers/Scheduling/CronSchedule.cs

```diff
@@ -60,10 +47,12 @@ public override string ToString()
         internal static bool TryCreate(string cronExpression, out CronSchedule cronSchedule)
         {
             cronSchedule = null;
```

Comment:
With the previous implementation, if `cronExpression` was `null`, that would just result in `TryCreate` returning `false` (not super clear, and a bit fragile as it just relied on the `TryParse` behavior), but with this change, it will result in a `null` reference exception when it hits the `Split` in the new `HasSeconds` method.

Can we update this to maintain the same behavior and return `false` when a `null` expression is passed in?

Reply:
Updated to replicate behaviour and wrapped in tests.